src/core/src/OmniTileset.cpp

```diff
@@ -280,6 +280,10 @@ TilesetStatistics OmniTileset::getStatistics() const {
 }
 
 void OmniTileset::reload() {
+    if (_renderResourcesPreparer != nullptr) {
+        _renderResourcesPreparer->detachTileset();
+    }
```

Comment:
This was something that I missed in https://github.com/CesiumGS/cesium-omniverse/pull/416.

If the cesium-native tileset is destroyed (like when a tileset reload occurs) we need to tell the render resource preparer to detach its tileset reference.